### PR TITLE
F3-LINE cyborg updates attempt 2 [Modularish]

### PR DESCRIPTION
**Pull request by pixelkitty286**
## About The Pull Request

This attempt two of https://github.com/Bubberstation/Bubberstation/pull/1636

Makes the sprite file resolutions from 64 x 32 to 32 x 32 for cat borgs

Makes 32 x 32 quadruped cybrogs into it's own borg type with it's own quadruped module of it's own.

Allows only station standard cat borgs to be hold able, non standard can not be held.

Adds code for cyborgs to indicate when they are being held.

Makes cyborg that are being held unable to interact items around them, they can still break free any time.

This does not allow cyborgs with shape shifting to be picked up it only allows the REAL model to be held.

A slight change from the last PR is the addition of a cargo variant of cat borg.

## Why It's Good For The Game

I've always wondered why can't I make cat borgs hold able and also make it so it don't have sweeping mechanical changes to the game so I came up with a way to allow them to be held disabling their abilities to interact with much wile being in a held state and being almost unbaggable.

Cargo cat borg was added since there was no variant.

Making the sprites 32 x 32 means they won't look so offsetted when using things quite as often.

Cent Com don’t want you to know this but the the cat borgs on space station 13 are free you can take them home I have 458 cat borgs at home.

## Proof Of Testing

dreamseeker_wBPMde6WO5

dreamseeker_xTFBOmZHyJ

sc19jFbsn7

dreamseeker_mmNECEkkUe

## Changelog

:cl:
add: holding code for cyborgs
del: Old 32x32 F3-LINE sprites
image: 32x32 F3-LINE sprites and a cargo variant
code: preventive flags to indicate when held
admin: Two robot traits added to the trait list
/:cl:

### CODE REVIEW

**Review by projectkepler-ru — APPROVED** [review]
Quite frankly minimal player-facing concerns from this

File: code/_onclick/cyborg.dm [hunk 1]
```diff
@@ -14,7 +14,7 @@
 	if(check_click_intercept(params,A))
 		return
 
-	if(stat || (lockcharge) || IsParalyzed() || IsStun())
+	if(stat || (lockcharge) || IsParalyzed() || IsStun() || (mob_in_holder)) //BUBBER EDIT - makes it so cyborgs that are holdable can not interact wile being held
```
Comment (Majkl-J) [inline code/_onclick/cyborg.dm:17]: Can't we just check with `istype(loc, /obj/item/clothing/head/mob_holder)` ?

**Review by Majkl-J — CHANGES_REQUESTED** [review]

**Review by Majkl-J — CHANGES_REQUESTED** [review]
Looks good otherwise

File: modular_zubbers/code/modules/borgs/code/robot_upgrade.dm [hunk 2]
```diff
@@ -60,3 +60,39 @@
 	items_to_add = list(/obj/item/storage/part_replacer/bluespace)
 	items_to_add -= list(/obj/item/storage/part_replacer)
 
+
+/// Quadruped special module
+/obj/item/borg/upgrade/squadrupedmodule
+	name = "small quadruped borg affection module"
+	desc = "A module that upgrades the ability of small quadruped borgs to display affection."
+	icon_state = "cyborg_upgrade3"
```
[the remark this answers is not in the document]
Comment (pixelkitty286) [reply modular_zubbers/code/modules/borgs/code/robot_upgrade.dm:68]: I tried this before and it ended up just duping the items and giving it to the unintended modules

**Comment by pixelkitty286** [comment]
fuck

File: modular_zubbers/code/modules/research/techweb/all_nodes.dm [hunk 3]
```diff
@@ -58,6 +58,12 @@
 	)
 	return ..()
 
+/datum/techweb_node/cyborg/New()
+	. = ..()
+	design_ids += list(
+		"saffection_module",
+	)
+
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/research/techweb/all_nodes.dm:66]:
```suggestion
```

**Review by Majkl-J — CHANGES_REQUESTED** [review]
Get rid of this and it should be ready to merge

**Comment by pixelkitty286** [comment]
> Get rid of this and it should be ready to merge

HUUUUUUUUUUUUUNNNNNNNNNNNNNGGGGG HOW HOW HOW DID I FOR GET THAT?!

one second

**Review by Majkl-J — APPROVED** [review]

**Pull request merged**